For Advanced/090.md

```diff
@@ -0,0 +1,45 @@
+# RDC Automated
+
+Take the script from the previous lesson and enhance it with some more UI actions to get a sample of a fully automatic configuration of the Remote Desktop Connection:
+
+```G1ANT
+program mstsc
```

Comment:
If you write long blocks of code separate groups of commands by empty lines.

Reply:
OK, but it's a bit hard to define separate groups here.